rust/oak_derive/Cargo.toml

```diff
@@ -13,3 +13,5 @@ syn = { version = "*", features = ["extra-traits"]}
 
 [dev-dependencies]
 oak = { path = "../oak" }
+oak_log = { path = "../oak_log" }
+log = { version = "^0.4.5", features = ["std"] }
```

Comment:
I think we should relax this too, maybe `0.4`? unless we really depend on something specific that is in `0.4.5` but no earlier? Anyway up to you.